public/data/eddiejaoude.json

```
@@ -1,7 +1,7 @@
 {
   "name": "Eddie Jaoude",
   "type": "personal",
-  "bio": "Open Source DevRel | Founder of EddieHub",
+  "bio": "Open Source DevRel | Founder of EddieHub| GitHub Star",
```

Hmm, what do you think if we add an extra space here? 🙂
```suggestion
  "bio": "Open Source DevRel | Founder of EddieHub | GitHub Star",
```

---

thank you 👍